pkg/controllers/clusterinfo/capacity_controller.go

```diff
@@ -62,25 +60,16 @@ func (r *CapacityReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 	}
 
 	nodes := clusterInfo.Status.NodeList
-	cpuWorkerCapacity := *resource.NewQuantity(int64(0), resource.DecimalSI)
-	socketTotalCapacity := *resource.NewQuantity(int64(0), resource.DecimalSI)
 	socketWorkerCapacity := *resource.NewQuantity(int64(0), resource.DecimalSI)
-	coreTotalCapacity := *resource.NewQuantity(int64(0), resource.DecimalSI)
 	coreWorkerCapacity := *resource.NewQuantity(int64(0), resource.DecimalSI)
 	for _, node := range nodes {
-		socketTotalCapacity.Add(node.Capacity[resourceSocket])
-		coreTotalCapacity.Add(node.Capacity[resourceCore])
 		if isWorker(node) {
-			cpuWorkerCapacity.Add(node.Capacity[clusterv1.ResourceCPU])
+			coreWorkerCapacity.Add(node.Capacity[clusterv1.ResourceCPU])
```

Comment:
I know that CPU/Core are the same but why you usse the lusterv1.ResourceCPU] to fill the core? isn't it confusing? Don't we have a capacity for the Core?
Ok got it... this is to optimize the api calls and so improve performance.